### PR TITLE
135.0

### PR DESCRIPTION
**Pull request by Furtif**

### CODE REVIEW

File: proto/Networking/Requests/Messages/ChangeTeamMessage.proto [hunk 1]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package POGOProtos.Networking.Requests.Messages;
+
+import "POGOProtos/Inventory/Item/ItemId.proto";
+
+message ChangeTeamMessage {
+	.POGOProtos.Inventory.Item.ItemId item = 1;
+	.POGOProtos.Inventory.Item.ItemId team = 2;
```
Comment (niicojs) [inline proto/Networking/Requests/Messages/ChangeTeamMessage.proto]: This is a Team, not an Item

File: proto/Networking/Responses/GetPhotobombResponse.proto [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import "POGOProtos/Enums/PokemonId.proto";
+import "POGOProtos/Data/PokemonDisplay.proto";
+
+message GetPhotobombResponse {
+	Status status = 1;
+	.POGOProtos.Enums.PokemonId pokemon_id = 2;
+	double lat = 3;
+	double lng = 4;
+	string encounter_location = 5;
+	uint64 encounter_id = 6;
```
Comment (niicojs) [inline proto/Networking/Responses/GetPhotobombResponse.proto]: should be fixed64

**Review by niicojs — CHANGES_REQUESTED** [review]

File: proto/Networking/Requests/Messages/ChangeTeamMessage.proto (hunk 1 above)
Comment (Furtif) [reply proto/Networking/Requests/Messages/ChangeTeamMessage.proto]: ok but this need special item also is item team maybe item is ok

File: proto/Networking/Requests/Messages/ChangeTeamMessage.proto [hunk 3]
```diff
@@ -1,9 +1,9 @@
 syntax = "proto3";
 package POGOProtos.Networking.Requests.Messages;
 
-import "POGOProtos/Inventory/Item/ItemId.proto";
+import "POGOProtos/Enums/TeamColor.proto";
 
 message ChangeTeamMessage {
-	.POGOProtos.Inventory.Item.ItemId item = 1;
-	.POGOProtos.Inventory.Item.ItemId team = 2;
+	.POGOProtos.Enums.TeamColor item = 1;
```
Comment (niicojs) [inline proto/Networking/Requests/Messages/ChangeTeamMessage.proto]: First is item, second is team.
Or am I missing something?

**Review by niicojs — CHANGES_REQUESTED** [review]

File: proto/Networking/Requests/Messages/ChangeTeamMessage.proto (hunk 1 above)
Comment (Furtif) [reply proto/Networking/Requests/Messages/ChangeTeamMessage.proto]:
```proto
import "POGOProtos/Enums/TeamColor.proto";
import "POGOProtos/Inventory/Item/ItemId.proto";

message ChangeTeamMessage {
	.POGOProtos.Inventory.Item.ItemId item = 1;
	.POGOProtos.Enums.TeamColor team = 2;
}
```

File: proto/Networking/Requests/Messages/ChangeTeamMessage.proto (hunk 3 above)
Comment (Furtif) [reply proto/Networking/Requests/Messages/ChangeTeamMessage.proto]: this ? https://github.com/pogosandbox/pogo-protos/pull/6#discussion_r260477238

Comment (niicojs) [reply proto/Networking/Requests/Messages/ChangeTeamMessage.proto]: yes

**Review by niicojs — APPROVED** [review]

**Pull request merged**